src/osx/iphone/colour.mm

```diff
@@ -0,0 +1,201 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        src/osx/iphone/colour.mm
+// Purpose:     Conversions between UIColor and wxColour
+// Author:      Vadim Zeitlin
+// Created:     2015-11-26 (completely replacing the old version of the file)
+// Copyright:   (c) 2015 Vadim Zeitlin
```

Comment:
My name shouldn't be here.

Reply:
sorry, copied the headers over from the cocoa version ...